src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$color-black: #000;
+$color-grey: #ccc;
+$color-red: red;
+$color-blue: blue;
```

Comment:
Consider using a specific hex color code for blue instead of the generic 'blue'.